ZConfig/__init__.py

```diff
@@ -33,13 +33,17 @@
 """
 __docformat__ = "reStructuredText"
 
-version_info = (3, 0)
-__version__ = ".".join([str(n) for n in version_info])
+from ZConfig._compat import TextIO
+import ZConfig.loader
 
-from ZConfig.loader import loadConfig, loadConfigFile
-from ZConfig.loader import loadSchema, loadSchemaFile
+loadConfigFile = ZConfig.loader.loadConfigFile
```

Comment:
I'm assuming this change was for something that was imported but not used? Meaning it's meant as part of the public API for this module? How do you feel about using `__all__` for that purpose? (Things imported but not used, but listed in `__all__` don't generate warnings.)

Reply:
Exactly.

I'm good with using ``__all__``, but also consider it a riskier change, since it'll impact anyone using ``from ZConfig import *`` (which I'd never advocate!).

We can certainly consider moving to using ``__all__`` as a strategy for managing exports, but I'd rather make that a separate change.

Reply:
Sounds good to me.